src/serac/coefficients/coefficient_extensions.hpp

```diff
@@ -14,41 +14,170 @@
 
 #include <functional>
 #include <memory>
-
+#include <variant>
+#include <vector>
 #include "mfem.hpp"
 
 #include "serac/numerics/expr_template_ops.hpp"
-
+#include "serac/physics/utilities/finite_element_state.hpp"
```

Comment:
What is this included for? If we want to have the coefficients depend on the physics_utilities, that dependency should probably be propagated to the CMakeLists.